app/models/user.rb

```diff
@@ -14,7 +14,7 @@ class User < ActiveRecord::Base
   GENDERS = %w(
     female
     male
-    other
+    others
```

Comment:
@markets what do you think to change that? I try to solve this problem https://github.com/coopdevs/timeoverflow/pull/498#issuecomment-492976580

I think is enough to change it here and in ymls, I tested locally and will do it again in staging

Reply:
Yep, I think this is enough 👍